### ISSUE TITLE
editor feature

### ISSUE THREAD

**Issue by heartacker** [fjeremic/HexView.Wpf#1]
hi is there plan  to implete editor feature?

**Comment by fjeremic** [comment]
No. This is simply a control for viewing hex data, hence Hex**View**. The control was not designed with writing in mind.
